models/User.ts

```diff
@@ -81,12 +95,8 @@ export default class User implements KongUser, GovDeliveryUser {
     return `${this.organization}${this.lastName}`.replace(/\W/g, '');
   }
 
-  private toSlackString(): string {
-    const intro = `${this.lastName}, ${this.firstName}: ${this.email}\nDescription: ${this.description}\nRequested access to:\n`;
-    return this.apiList.reduce((m, api) => m.concat(`* ${api}\n`), intro);
-  }
-
   public get apiList(): string[] {
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
maybe this rule should be reviewed at a later point. seems very arbitrary and kind of counter to typescript getters

Reply:
Agreed

Reply:
This should probably just be a `readonly` property anyway